### PR TITLE
[MIG] sale_order_general_discount: Migration to 13.0 

### PR DESCRIPTION
**Pull request by maitaoriana**
The work was already done by this PR: #1124

### CODE REVIEW

**Comment by maitaoriana** [comment]
Hi @pedrobaeza, I made the changes requested by you on this PR #1124. This PR have the commits made by @sudhir-erpharbor and @acsonefho but with squash and pre-commit . Please, Could you review it? Thanks. 
Regards

**Review by acsonefho — APPROVED** [review]
Great!!

File: sale_order_general_discount/README.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+#. Also you can set a discount in a partner.
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/sale-workflow/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/sale-workflow/issues/new?body=module:%20sale_order_general_discount%0Aversion:%2012.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (sudhir-erpharbor) [inline sale_order_general_discount/README.rst:59]: LGTM. Just a small change. The version should be 13.0.

**Pull request merged**